components/concept/engine/src/main/java/mozilla/components/concept/engine/permission/PermissionRequest.kt

```diff
@@ -69,8 +69,6 @@ sealed class Permission(open val id: String? = "", open val desc: String? = "")
     @Parcelize
     data class ContentAudioOther(override val id: String? = "", override val desc: String? = "") : Permission(id)
     @Parcelize
-    data class ContentAutoplayMedia(override val id: String? = "", override val desc: String? = "") : Permission(id)
```

Comment:
This is causing something an issue with the other variants gecko-beta and stable as this is share between them.

```
> Task :support-utils:transformClassesAndResourcesWithPrepareIntermediateJarsForDebug
> Task :browser-engine-gecko:compileDebugKotlin FAILED
e: /build/android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/permission/GeckoPermissionRequest.kt: (56, 57): Unresolved reference: ContentAutoplayMedia
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':browser-engine-gecko:compileDebugKotlin'.
> Compilation error. See log for more details
```


```

> Task :support-utils:transformClassesAndResourcesWithPrepareIntermediateJarsForDebug
> Task :browser-engine-gecko-beta:compileDebugKotlin FAILED
e: /build/android-components/components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/permission/GeckoPermissionRequest.kt: (56, 57): Unresolved reference: ContentAutoplayMedia
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':browser-engine-gecko-beta:compileDebugKotlin'.
> Compilation error. See log for more details
```